generators/client/templates/angular/package.json.ejs

```diff
@@ -171,11 +174,22 @@
     "ngc": "ngc -p tsconfig.app.json",
     "cleanup": "rimraf <%= DIST_DIR %>",
     "clean-www": "rimraf <%= DIST_DIR %>app/{src,<%= BUILD_DIR %>}",
-    <%_ if (protractorTests) { _%>
-    "e2e": "protractor <%= TEST_SRC_DIR %>protractor.conf.js",
+<%_ if (protractorTests && !cypressTests) { _%>
+    "e2e": "<%= clientPackageManager %> run e2e:protractor",
+<%_ } else if ((cypressTests && !protractorTests) || (protractorTests && cypressTests)) {  _%>
```

Comment:
```suggestion
<%_ } else if (protractorTests && cypressTests) {  _%>
    "e2e": "<%= clientPackageManager %> run e2e:cypress && <%= clientPackageManager %> run e2e:protractor",
<%_ } else if (cypressTests && !protractorTests {  _%>
```

Reply:
after a short discussion with @pascalgrimaud we would to set cypress as default script if both cypress and protractor are choosen.

Reply:
Ok then, so I think we should not migrate to cypress at current tests.
While entities is not implemented at cypress.

Reply:
We have not implemented entities test yet because some entities test are commented in protractor and it is a bit tricky with entities which have relationships. WDYT ?

Reply:
@nassimerrahoui take your time to implement, but merging the way it is now would reduce coverage.
I would prefer to merge as soon as possible to reduce possibility of conflicts and migrate to cypress by default later.

Reply:
In fact, protractor entities tests seems not to be functional. I will add a todo in this PR to discuss about that.

Reply:
I agree with @mshima, we just need to define by default protactor and we can commit cypress has 'alfa' or 'beta'

Reply:
The real issue is that we need to test as much as possible the possibilities offered by JHipster with Cypress. The recent work prove that we can fail on specific options (e.g. session).

Maybe the best solution is to run both protractor and cypress on each build OR to add protractor specific daily builds. We could have all CI builds running with Cypress and protractor daily builds.

Reply:
So better to use the daily builds for this

Reply:
As @mshima said, let's add entities e2e with cypress in this PR.
Don't revert the change about default Cypress in the CI here

Protractor is a pain to maintain, I don't want this option by default anymore

Reply:
Entity tests done here [avdev4j/55](https://github.com/avdev4j/generator-jhipster/pull/55)

Reply:
It's only missing insertion and deletion.
They are important.
https://github.com/jhipster/generator-jhipster/blob/3ac9fb0f36492b5fe3e68c0c1bc822875e6c568c/generators/entity-client/templates/angular/src/test/javascript/e2e/entities/entity.spec.ts.ejs#L104-L213

This PR should provide an easier way to create data instead of current cypress. https://github.com/jhipster/generator-jhipster/pull/12253